docs/Public-api.md

```diff
@@ -0,0 +1,132 @@
+## pc (gpucsl/pc/pc.py)
+
+pc(
+    data: np.ndarray,
+    data_distribution: DataDistribution,
+    max_level: int,
+    alpha=0.05,
+    kernels=None,
+    is_debug: bool = False,
+    should_log: bool = False,
+    devices: List[int] = [0],
+    sync_device: int = None,
+    gaussian_correlation_matrix: np.ndarray = None,
+    discrete_max_memory_size=None,
+) -> PCResult
```

Comment:
Consider enclosing the code like
```
code;
```
Otherway, it is formatted quite strange in github

Reply:
Hmm, to be honest I wanted to reproduce the style used in the cupy docs (e.g. see https://docs.cupy.dev/en/stable/reference/generated/cupy.RawKernel.html). There it is formatted as in the final rendered file.

The idea is that the function itself is not that important, but the parameters you call it with and they are explained in a list below it.